src/main/java/org/grpc/entities/Employee.java

```diff
@@ -0,0 +1,79 @@
+package org.grpc.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString
+public class Employee {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
ty si banan

Reply:
It's a method to automatically generate Ids so that JPA will know how to handle them, if an entity is created. GenerationType.AUTO means increment by 50 by default, sometimes (no idea when tho) by one